server/lib/routes/get-index.js

```diff
@@ -26,7 +27,8 @@ module.exports = function (config) {
     oAuthClientId: CLIENT_ID,
     oAuthUrl: OAUTH_SERVER_URL,
     profileUrl: PROFILE_SERVER_URL,
-    release: RELEASE
+    release: RELEASE,
+    scopedKeysEnabled: SCOPED_KEYS_ENABLED
```

Comment:
Do we have a test for this module?

Reply:
Yea, we can add this new config prop into an existing test 👍